data/src/main/java/com/example/data/datasource/PostsRemoteDataSourceImpl.kt

```diff
@@ -19,13 +21,14 @@ internal class PostsRemoteDataSourceImpl @Inject constructor(
     private val postsApi: PostsApi,
     private val postsMapper: PostRemoteMapper,
     private val commentsMapper: BaseRemoteMapper<List<CommentsRemote>, List<Comments>>,
-    private val userMapper: BaseRemoteMapper<UserRemote, User>
+    private val userMapper: BaseRemoteMapper<UserRemote, User>,
+    @IoDispatcher private val ioDispatcher: CoroutineDispatcher
```

Comment:
The only comment I would make is to change the name of the parameter to not tie it to an ioDispatcher